### PR TITLE
Gidk 1hbj4z scroll issue

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/settings/create-address/create-address.component.html [hunk 1]
```diff
@@ -13,233 +13,235 @@ <h3 *ngIf="addressConfiguration.type === 'editWarehouse'">Edit Warehouse</h3>
...
-                            <div class="row">
-                                <div *ngIf="addressConfiguration?.tax?.name" class="col-sm-4 form-group">
-                                    <label>{{addressConfiguration?.tax?.name}}</label>
-                                    <input
-                                        [class.error-box]="addressForm?.get('taxNumber')?.touched && addressForm?.get('taxNumber')?.invalid"
-                                        type="text" (keyup)="getStateCode(states, $event)" class="form-control"
-                                        formControlName="taxNumber"
-                                        [attr.placeholder]="'Enter '+ addressConfiguration?.tax?.name">
+                            <br>
+                            <div class="form_bg clearfix">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/create-address/create-address.component.html:23]: Avoid using snake_case in CSS classes

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/webpack.partial.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('http://localhost:3000/'),
-            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
+            'ApiUrl': JSON.stringify('https://apidev.giddh.com/'),
```
Comment (prateek-giddh) [inline apps/web-giddh/webpack.partial.js:13]: Revert these changes

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]
